enso/experiment/__init__.py

```diff
@@ -162,8 +167,32 @@ def _run_sub_experiment(
 
     def _run_experiment(self, dataset_name, current_setting, experiments):
         """Responsible for running all experiments specified in config."""
-        results = pd.DataFrame(columns=self.columns)
         dataset = self._load_dataset(dataset_name, current_setting.get("Featurizer"))
+        if EXPERIMENT_PARAMS:
+            exp_params = deepcopy(EXPERIMENT_PARAMS)
```

Comment:
Ah cool -- I suppose means your default arg would have been fine but it's still probably worth changing to None anyhow.

Reply:
Oh that's what you meant, gotcha